bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/pages/ExtensionsPage.java

```diff
@@ -204,20 +194,47 @@ protected WebMarkupContainer createDialog(String id) {
 		pwstrengthSample();
 	}
 
-	private void pwstrengthSample() {
+	private void dropzoneFileUpload()
+	{
+		DropZoneFileUpload dropZoneFileUpload = new DropZoneFileUpload("dropzone")
+		{
+			private static final long serialVersionUID = 1L;
+
+			@Override
+			protected void onUpload(AjaxRequestTarget target)
+			{
+				try
+				{
+					ServletWebRequest webRequest = (ServletWebRequest)getRequest();
+					MultipartServletWebRequest multiPartRequest = webRequest.newMultipartWebRequest(
+						Bytes.bytes(getMaxFileSize() * 1000000), "ignored");
+					multiPartRequest.parseFileParts();
+					Map<String, List<FileItem>> files = multiPartRequest.getFiles();
```

Comment:
Okay - we can do this - I also thought about it but I decided to handle it like in wicket forms. I am fine with this, too.